main.sh

```diff
@@ -80,8 +81,6 @@ cluster_up() {
 cluster_down() {
 	"$CLUSTER_PROVIDER"_down
 	rm "$KUBECONFIG_ROOT_DIR/$KEPLER_KUBECONFIG"
-	export KUBECONFIG="$KUBECONFIG_ROOT_DIR"/config
```

Comment:
should we keep this ? if the cluster is down, we don't have reason to keep config file?

Reply:
Missed it😅. Yes we can remove it